jobsdb/jobsdb.go

```diff
@@ -1487,20 +1490,6 @@ func (jd *HandleT) postDropDs(ds dataSetT) {
 	jd.isStatDropDSPeriodInitialized = true
 }
 
-func (jd *HandleT) invalidateCache(ds dataSetT) {
-	// Trimming pre_drop from the table name
-	if strings.HasPrefix(ds.JobTable, preDropTablePrefix) {
-		parentDS := dataSetT{
-			JobTable:       strings.ReplaceAll(ds.JobTable, preDropTablePrefix, ""),
-			JobStatusTable: strings.ReplaceAll(ds.JobStatusTable, preDropTablePrefix, ""),
-			Index:          ds.Index,
-		}
-		jd.dropDSFromCache(parentDS)
-	} else {
-		jd.dropDSFromCache(ds)
-	}
-}
-
```

Comment:
**Note:** we no longer need to do all this trickery, since we are now only interested about the dataset's index, not the full dataSetT struct.